### ISSUE TITLE
Check buttonReverse property in reset method

### ISSUE THREAD

**Issue by fabien-d** [fabien-d/alertify.js#91]
The reset method in the `addListeners` method needs to take the `buttonReverse` property into account when resetting the focus. If button are reversed, btnOK and btnCancel should be reversed.

``` js
reset = function (event) {
    if (hasInput) input.focus();
    else if (hasCancel) btnCancel.focus();
    else btnOK.focus();
};
```

**Comment by fabien-d** [comment]
Merged. Issue resolved in `0.3.9`. Branch `0.3` of repo or can be downloaded for the [demo site](http://fabien-d.github.io/alertify.js/).

Leaving opening because it's still an issue for `0.4`.

**Comment by fabien-d** [comment]
`0.4` is deprecated - closing
